templates/example_cell_clustering.ipynb

```diff
@@ -11,13 +11,15 @@
    "cell_type": "markdown",
```

Comment:
<p>Not super important but I think the default for 'pixel_cluster_col' should be 'pixel_meta_cluster' - that will be more commonly used.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/460/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Ah yes, I forgot to change this back. I wanted to test with <code>pixel_som_cluster</code>  earlier to verify that anyone who insisted on using this for cell clustering would not run into any errors.</p>